### ISSUE TITLE
Dropout Layer

### ISSUE THREAD

**Issue by MikeInnes** [FluxML/Flux.jl#88]
It would be useful to have a standard dropout layer. Should be easy to implement.

**Comment by jekbradbury** [comment]
Could do it the “right” way or could do it the [fast way](https://twitter.com/scottgray76/status/917497407843188737)

**Comment by iblis17** [comment]
I think the dropout layer also need two differnet phase on training and testing. This situation is same as BatchNorm.
